spec/features/admin_dashboard_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+RSpec.feature "Submit application with minimal information" do
+  scenario "The stats are accurate", javascript: true do
+    page.driver.browser.current_session.basic_authorize("admin", "password")
+    5.times { create(:snap_application, signed_at: nil) }
+    5.times { create(:snap_application, :emailed, signed_at: nil) }
+    10.times { create(:snap_application, signed_at: 5.days.ago) }
+    5.times { create(:snap_application, :faxed, signed_at: 5.days.ago) }
+    visit admin_root_path
+    expect(page).to have_content "25 Created"
+    expect(page).to have_content "15 Signed"
+    expect(page).to have_content "5 Emailed"
+    expect(page).to have_content "5 Faxed"
```

Comment:
since creating records slows down the specs, what do you think is the minimal number of records we could create here to confirm that the numbers are accurate?

Reply:
Yea, I wanted to get over the 20 boundary that would cause pagination to kick in, but 21 should be fine instead of 25.

Reply:
in that case, what I usually do is stub the pagination numbers for the tests so I don't need quite as many records

Reply:
ooooh that's a great idea.... I should have thought of that before merge. This test takes ~1s on my machine, at some point we could do a refactor pass to optimize the test suite in a later ticket?

Reply:
yes of course! wanted to get this line of thinking on our radar bc minimal record creation is an easy win for a fast test suite, but no need to change anything now